### PR TITLE
Adds zenodo metadata file

### PR DESCRIPTION
**Pull request by dpshelio**
### Description

Adds a `.zenodo.json` metadata file with information about the authors and the project title, license and other info.

I've tried to add updated information to everyone listed there, but I'm sure there's others that could/should be there. The order is provided as number of contributions as Zenodo see it. We can change it if we want. 

Things I don't know:
 - In newer release would zenodo add more authors to the list or it's fixed from this?
 - I've removed some fields like version because I believed zenodo will do automatically from the release if one is not set.
 - will my name with accents work?

Please, if your name is below add a suggestion with your orcid, affiliation and name and  I will update it.

@sunpy/sunpy-board

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            "affiliation": "Aperio Software",
+            "name": "Nabil Freij"
+        },
+        {
+            "affiliation": "@google",
+            "name": "Florian Mayer"
+        },
+        {
+            "affiliation": "National Institutes of Health",
+            "name": "Keith Hughitt"
```
Comment (dpshelio) [inline .zenodo.json]: @khughitt - do you have an orcid? is that affiliation correct?

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "affiliation": "1 - NASA Goddard Space Flight Center. 2 - Catholic University of America",
+            "name": "Daniel F. Ryan"
```
Comment (dpshelio) [inline .zenodo.json]: @DanRyanIrish ORCID?

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Simon Liedtke"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Albert Y. Shih"
```
Comment (dpshelio) [inline .zenodo.json]: @ayshih Orcid?

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Pritish Chakraborty"
+        },
+        {
+            "affiliation": "IIT Indore",
+            "name": "Vishnunarayan K I"
+        },
+        {
+            "name": "Andrew R. Inglis"
```
Comment (dpshelio) [inline .zenodo.json]: @aringlis ORCID? Affiliation?

File: .zenodo.json [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "affiliation": "IIIT Hyderabad",
+            "name": "Punyaslok Pattnaik"
+        },
+        {
+            "affiliation": "DIRAC Institute, UW",
+            "name": "Brigitta M. Sipocz",
+            "orcid": "0000-0002-3713-6337"
```
Comment (dpshelio) [inline .zenodo.json]: @bsipocz is this correct?

File: .zenodo.json [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "name": "Rishabh Sharma"
+        },
+        {
+            "name": "Drew Leonard"
+        },
+        {
+            "affiliation": "Virginia Tech",
+            "name": "Russell J. Hewett",
+            "orcid": "0000-0001-8944-4705"
```
Comment (dpshelio) [inline .zenodo.json]: @rhewett is this correct?

File: .zenodo.json [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Alex-Ian-Hamilton"
```
Comment (dpshelio) [inline .zenodo.json]: @Alex-Ian-Hamilton Orcid? affiliation?

File: .zenodo.json [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "affiliation": "Imperial College London",
+            "name": "David Stansby",
+            "orcid": "0000-0002-1365-1908"
```
Comment (dpshelio) [inline .zenodo.json]: @dstansby is this correct?

File: .zenodo.json [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "affiliation": "@kossiitkgp",
+            "name": "Nitin Choudhary"
+        },
+        {
+            "name": "Ankit Kumar"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Laura Hayes"
```
Comment (dpshelio) [inline .zenodo.json]: @hayesla Congratulations on your new paper!! :tada:  - how come you don't have an ORCID?

File: .zenodo.json [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Sudarshan Konge"
+        },
+        {
+            "name": "mdmueller"
+        },
+        {
+            "affiliation": "NASA/GSFC",
+            "name": "Michael Kirk"
```
Comment (dpshelio) [inline .zenodo.json]: @MSKirk ORCID? is the affiliation correct?

File: .zenodo.json [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+            "orcid": "0000-0003-4217-4642"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Steven Christe",
+            "orcid": "0000-0001-6127-795X"
+        },
+        {
+            "affiliation": "Aperio Software",
+            "name": "Nabil Freij"
```
Comment (dpshelio) [inline .zenodo.json]: @nabobalis ORCID?

File: .zenodo.json [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "affiliation": "@google",
+            "name": "Florian Mayer"
```
Comment (dpshelio) [inline .zenodo.json]: @segfaulthunter orcid? affiliation is correct?

File: .zenodo.json [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "affiliation": "National Institutes of Health",
+            "name": "Keith Hughitt"
+        },
+        {
+            "affiliation": "1 - NASA Goddard Space Flight Center. 2 - Catholic University of America",
+            "name": "Daniel F. Ryan"
+        },
+        {
+            "name": "Simon Liedtke"
```
Comment (dpshelio) [inline .zenodo.json]: @derdon affiliation? ORCID?

File: .zenodo.json [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+            "affiliation": "University College London",
+            "name": "David Pérez-Suárez",
+            "orcid": "0000-0003-0784-6909"
+        },
+        {
+            "name": "Pritish Chakraborty"
+        },
+        {
+            "affiliation": "IIT Indore",
+            "name": "Vishnunarayan K I"
```
Comment (dpshelio) [inline .zenodo.json]: @vn-ki orcid? affiliation correct?

File: .zenodo.json [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Albert Y. Shih"
+        },
+        {
+            "affiliation": "University College London",
+            "name": "David Pérez-Suárez",
+            "orcid": "0000-0003-0784-6909"
+        },
+        {
+            "name": "Pritish Chakraborty"
```
Comment (dpshelio) [inline .zenodo.json]: @PritishC ORCID? Affiliation?

File: .zenodo.json [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "affiliation": "IIT Indore",
+            "name": "Vishnunarayan K I"
+        },
+        {
+            "name": "Andrew R. Inglis"
+        },
+        {
+            "affiliation": "IIIT Hyderabad",
+            "name": "Punyaslok Pattnaik"
```
Comment (dpshelio) [inline .zenodo.json]: @Punyaslok ORCID? affiliation correct?

File: .zenodo.json [hunk 17]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "affiliation": "DIRAC Institute, UW",
+            "name": "Brigitta M. Sipocz",
+            "orcid": "0000-0002-3713-6337"
+        },
+        {
+            "name": "Rishabh Sharma"
+        },
+        {
+            "name": "Drew Leonard"
```
Comment (dpshelio) [inline .zenodo.json]: @SolarDrew ORCID? affiliation?

File: .zenodo.json [hunk 18]
```diff
@@ -0,0 +1,130 @@
...
+            "affiliation": "IIIT Hyderabad",
+            "name": "Punyaslok Pattnaik"
+        },
+        {
+            "affiliation": "DIRAC Institute, UW",
+            "name": "Brigitta M. Sipocz",
+            "orcid": "0000-0002-3713-6337"
+        },
+        {
+            "name": "Rishabh Sharma"
```
Comment (dpshelio) [inline .zenodo.json]: @gunner272 ORCID? affiliation?

File: .zenodo.json [hunk 19]
```diff
@@ -0,0 +1,130 @@
...
+            "name": "Alex-Ian-Hamilton"
+        },
+        {
+            "affiliation": "Imperial College London",
+            "name": "David Stansby",
+            "orcid": "0000-0002-1365-1908"
+        },
+        {
+            "affiliation": "International Insititute of Information Technology Bhubaneswar",
+            "name": "Asish Panda"
```
Comment (dpshelio) [inline .zenodo.json]: @kaichogami ORCID? affiliation correct?

File: .zenodo.json [hunk 20]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Matt Earnshaw"
```
Comment (dpshelio) [inline .zenodo.json]: @mattearnshaw ORCID? affiliation correct?

File: .zenodo.json [hunk 21]
```diff
@@ -0,0 +1,130 @@
...
+            "name": "Prateek Chanda"
+        },
+        {
+            "name": "Md Akramul Haque"
+        },
+        {
+            "name": "Sudarshan Konge"
+        },
+        {
+            "name": "mdmueller"
```
Comment (dpshelio) [inline .zenodo.json]: @mdmueller affiliation? Orcid?

File: .zenodo.json [hunk 22]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "name": "Ankit Kumar"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Laura Hayes"
+        },
+        {
+            "affiliation": "@sunpy  @fossasia @apache",
+            "name": "Prateek Chanda"
```
Comment (dpshelio) [inline .zenodo.json]: @prateekiiest affiliation? Orcid?

File: .zenodo.json [hunk 23]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Md Akramul Haque"
```
Comment (dpshelio) [inline .zenodo.json]: @Akram9 affiliation? Orcid?

File: .zenodo.json [hunk 24]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Sudarshan Konge"
```
Comment (dpshelio) [inline .zenodo.json]: @sudk1896 affiliation? Orcid?

File: .zenodo.json [hunk 25]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Matt Earnshaw"
+        },
+        {
+            "affiliation": "@kossiitkgp",
+            "name": "Nitin Choudhary"
+        },
+        {
+            "name": "Ankit Kumar"
```
Comment (dpshelio) [inline .zenodo.json]: @ankitkmr affiliation? Orcid?

File: .zenodo.json [hunk 26]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "affiliation": "International Insititute of Information Technology Bhubaneswar",
+            "name": "Asish Panda"
+        },
+        {
+            "name": "Matt Earnshaw"
+        },
+        {
+            "affiliation": "@kossiitkgp",
+            "name": "Nitin Choudhary"
```
Comment (dpshelio) [inline .zenodo.json]: @nitinkgp23 affiliation? Orcid?

File: .zenodo.json [hunk 27]
```diff
@@ -0,0 +1,130 @@
...
+    "creators": [
+        {
+            "affiliation": "1 - DKIST. 2 - Aperio Software",
+            "name": "Stuart Mumford",
+            "orcid": "0000-0003-4217-4642"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Steven Christe",
+            "orcid": "0000-0001-6127-795X"
```
Comment (dpshelio) [inline .zenodo.json]: @ehsteve correct?

File: .zenodo.json [hunk 28]
```diff
@@ -0,0 +1,130 @@
+{
+    "description": "The community-developed, free and open-source solar data analysis environment for Python.",
+    "license": "BSD-2-Clause",
+    "title": "SunPy",
+    "upload_type": "software",
+    "creators": [
+        {
+            "affiliation": "1 - DKIST. 2 - Aperio Software",
+            "name": "Stuart Mumford",
+            "orcid": "0000-0003-4217-4642"
```
Comment (dpshelio) [inline .zenodo.json]: @Cadair correct?

File: .zenodo.json (hunk 11 above)
Comment (nabobalis) [reply .zenodo.json]: Not a clue. Should I have one? 

Probably change the affiliation to none.

File: .zenodo.json (hunk 5 above)
Comment (bsipocz) [reply .zenodo.json]: ORCID is correct, but how correct we should be with the affiliation? I wouldn't think UW is unambiguous enough.

File: .zenodo.json [hunk 29]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "name": "Andrew R. Inglis"
+        },
+        {
+            "affiliation": "IIIT Hyderabad",
+            "name": "Punyaslok Pattnaik"
+        },
+        {
+            "affiliation": "DIRAC Institute, UW",
+            "name": "Brigitta M. Sipocz",
```
Comment (bsipocz) [inline .zenodo.json]:
```suggestion
            "name": "Brigitta M. Sipőcz",
```

File: .zenodo.json [hunk 30]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Andrew R. Inglis"
+        },
+        {
+            "affiliation": "IIIT Hyderabad",
+            "name": "Punyaslok Pattnaik"
+        },
+        {
+            "affiliation": "DIRAC Institute, UW",
```
Comment (bsipocz) [inline .zenodo.json]:
```suggestion
            "affiliation": "DIRAC Institute, Department of Astronomy, University of Washington",
```

File: .zenodo.json (hunk 22 above)
Comment (prateekiiest) [reply .zenodo.json]: Affiliation : IIEST Shibpur
Orcid: https://orcid.org/0000-0002-7068-2866

File: .zenodo.json [hunk 31]
```diff
@@ -0,0 +1,130 @@
+{
+    "description": "The community-developed, free and open-source solar data analysis environment for Python.",
+    "license": "BSD-2-Clause",
+    "title": "SunPy",
+    "upload_type": "software",
+    "creators": [
+        {
+            "affiliation": "1 - DKIST. 2 - Aperio Software",
```
Comment (Cadair) [inline .zenodo.json]:
```suggestion
            "affiliation": "1 - University of Sheffield. 2 - National Solar Observatory / DKIST. 3 - Aperio Software",
```

File: .zenodo.json (hunk 28 above)
Comment (Cadair) [reply .zenodo.json]: Yes, apart from correction to affiliations.

File: .zenodo.json [hunk 32]
```diff
@@ -0,0 +1,130 @@
...
+            "name": "Stuart Mumford",
```
Comment (Cadair) [inline .zenodo.json]:
```suggestion
            "name": "Stuart J. Mumford",
```

**Comment by Cadair** [comment]
We need to merge this before 1.0 really to make sure the 1.0 record is up to scratch.

File: .zenodo.json (hunk 20 above)
Comment (mattearnshaw) [reply .zenodo.json]: https://orcid.org/0000-0001-8236-2811
blank affiliation is fine.

File: .zenodo.json (hunk 8 above)
Comment (dstansby) [reply .zenodo.json]: 👍

File: .zenodo.json (hunk 17 above)
Comment (SolarDrew) [reply .zenodo.json]: 0000-0001-5270-7487, Aperio Software

File: .zenodo.json [hunk 33]
```diff
@@ -0,0 +1,130 @@
...
+            "orcid": "0000-0003-4217-4642"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Steven Christe",
+            "orcid": "0000-0001-6127-795X"
+        },
+        {
+            "affiliation": "Aperio Software",
```
Comment (dpshelio) [inline .zenodo.json]: What about?
```suggestion
            "affiliation": "Independent",
```

File: .zenodo.json (hunk 11 above)
Comment (dpshelio) [reply .zenodo.json]: Independent?
ORCID is a nice barcode you can tattoo on you ;)

File: .zenodo.json (hunk 17 above)
Comment (dpshelio) [inline .zenodo.json]:
```suggestion
            "name": "Drew Leonard",
            "affiliation": "Aperio Software",
            "orcid": "0000-0001-5270-7487"
```

File: .zenodo.json (hunk 20 above)
Comment (dpshelio) [reply .zenodo.json]:
```suggestion
            "name": "Matt Earnshaw",
            "orcid": "0000-0001-8236-2811"
```

File: .zenodo.json [hunk 34]
```diff
@@ -0,0 +1,130 @@
...
+        },
+        {
+            "name": "Ankit Kumar"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Laura Hayes"
+        },
+        {
+            "affiliation": "@sunpy  @fossasia @apache",
```
Comment (dpshelio) [inline .zenodo.json]:
```suggestion
            "affiliation": "IIEST Shibpur",
            "orcid": "0000-0002-7068-2866",
```

Comment (prateekiiest) [reply .zenodo.json]: Sorry for this unnecessary notification @dpshelio .
I thought that this was created before in an automated fashion, scraping github user's affiliation using the GitHub API, I guess.

**Comment by nabobalis** [comment]
@dpshelio Is this the full contributor list from github or just some of them?

File: .zenodo.json (hunk 11 above)
Comment (nabobalis) [reply .zenodo.json]: I created a blank one https://orcid.org/0000-0002-6253-082X

File: .zenodo.json (hunk 33 above)
Comment (nabobalis) [reply .zenodo.json]: Actually if this for 1.0, University of Leicester should be ok.

File: .zenodo.json (hunk 3 above)
Comment (ayshih) [reply .zenodo.json]: 0000-0001-6874-2594

File: .zenodo.json [hunk 35]
```diff
@@ -0,0 +1,134 @@
...
+        {
+            "affiliation": "1 - NASA Goddard Space Flight Center. 2 - Catholic University of America",
+            "name": "Daniel F. Ryan"
+        },
+        {
+            "name": "Simon Liedtke"
+        },
+        {
+            "affiliation": "NASA Goddard Space Flight Center",
+            "name": "Albert Y. Shih"
```
Comment (ayshih) [inline .zenodo.json]:
```suggestion
            "name": "Albert Y. Shih",
            "orcid": "0000-0001-6874-2594"
```

**Comment by Cadair** [comment]
@dpshelio Are we going to want to merge this whenever we are ready to go for 1.0? Is it better to merge this even if we aren't sure on all the details? Can you make the changes needed on here?

File: .zenodo.json (hunk 27 above)
Comment (ehsteve) [reply .zenodo.json]: yes. See https://github.com/heliophysicsPy/standards/blob/master/.zenodo.json#L54

**Comment by Cadair** [comment]
I have merged this, but if you got pinged on this PR and you want to update your details please either leave a comment or send a PR updating the file :smile:

File: .zenodo.json (hunk 9 above)
Comment (hayesla) [reply .zenodo.json]: Thanks @dpshelio  - oh I do I think https://orcid.org/0000-0002-6835-2390

File: .zenodo.json (hunk 10 above)
Comment (MSKirk) [reply .zenodo.json]: ORCID: 0000-0001-9874-1429
affiliation: 1 - NASA Goddard Space Flight Center. 2 - Catholic University of America

**Pull request merged**